src/classes/UTIL_RecordTypes_TEST.cls

```diff
@@ -40,47 +40,40 @@ private class UTIL_RecordTypes_TEST {
     // TEST
     static testmethod void testRecTypes() {
         // try bogus values
-        Id reallyBogus = UTIL_RecordTypes.GetRecordTypeId('Bogus', 'Bogus');
-        Id bogus = UTIL_RecordTypes.GetRecordTypeId('Opportunity', 'Bogus');
-        bogus = UTIL_RecordTypes.GetRecordTypeId('Contact', 'Bogus');
+        Id bogus = UTIL_RecordTypes.GetRecordTypeId(Opportunity.sObjectType, 'Bogus');
+        bogus = UTIL_RecordTypes.GetRecordTypeId(Contact.sObjectType, 'Bogus');
         bogus = UTIL_RecordTypes_API.GetRecordTypeId('Campaign', 'Bogus');
         
         // try all the functions
         set<Id> oppRtSet = UTIL_RecordTypes_API.GetRecordTypeIdSet('Opportunity', (new set<string>{'Bogus', 'Master', 'Grant'}));
-        Map<String, Id> oppTypeMap = UTIL_RecordTypes.GetRecordTypeIdMap('Opportunity');
```

Comment:
I dropped this line because I removed the `GetRecordTypeIdMap` method, since it was not being used by any external code